package.json

```diff
@@ -3,9 +3,6 @@
   "version": "1.0.0",
   "repository": "https://github.com/wagtail/wagtail",
   "private": true,
-  "engines": {
-    "node": ">= 4"
-  },
```

Comment:
It turns out this property is only respected when the package is installed as a dependency – so useless for our purposes.